layouts/partials/cmd-table.html

```diff
@@ -13,6 +13,7 @@
       {{ if in (slice         
         (and (eq $pageLang "en") (eq .locale "en_US"))
         (and (eq $pageLang "zh-cn") (eq .locale "zh_CN"))
+        (and (eq $pageLang "es-us") (eq .locale "es_US"))
```

Comment:
In the aegisub local (`data/commands_32.yaml`) only `locale: es` exists, not `locale: es_us`. Do you think those translations can be used?
```suggestion
        (and (eq $pageLang "es-us") (eq .locale "es"))
```
If you want the command to actually be displayed, you will also need to copy the `docs\latest\Commands.md` file over and remove the menu overwrite.

Reply:
Did so in [aed0ea7](https://github.com/Aegisub/aegisite/pull/32/commits/aed0ea7311bad0999ff4295d6870534c1d597c24)